src/sage/rings/function_field/jacobian_hess.py

```diff
@@ -606,7 +606,7 @@ def _element_constructor_(self, x):
         """
         Construct an element of ``self`` from ``x``.
 
-        If ``x`` is an effective divisor, then it is assumed to of
+        If ``x`` is an effective divisor, then it is assumed to be of
```

Comment:
```suggestion
        If ``x`` is an effective divisor, then it must be of
```
I think this is more clear, since an error is thrown if this condition is violated.